tests/unit_test/channels/mail_scheduler_test.py

```
@@ -0,0 +1,139 @@
+
+import pytest
+from unittest.mock import patch
+import os
+from kairon import Utility
+from kairon.exceptions import AppException
+
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff (0.8.0)

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)